app/models/user.rb

```diff
@@ -3,4 +3,14 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable, :confirmable,
          :recoverable, :rememberable, :trackable, :validatable
+
+  has_many :wikis
+
+  after_initialize :set_defaults
+
+  private
+
+  def set_defaults
```

Comment:
`set_defaults` is a bit vague - I'd recommend a more descriptive method, like `set_role`